app/models/atmosphere/appliance_set.rb

```diff
@@ -70,5 +69,34 @@ def production?
     def development?
       appliance_set_type.development?
     end
+
+    private
+
+    def self.devel_clew_includes
+      hsh = basic_clew_includes.tap do |hsh|
```

Comment:
Shadowing outer local variable - `hsh`.<br>Useless assignment to variable - `hsh`.